docs/stencil-docs/developing-further/customizing-emails.md

```diff
@@ -0,0 +1,22 @@
+# BigCommerce Email Templates
+
+<div class="otp" id="no-index">
+
+### On This Page
+- [BigCommerce Email Templates](#bigcommerce-email-templates)
+- [Accessing and Editing Email Templates](#accessing-and-editing-email-templates)
+
+</div>
+
+
+## BigCommerce Email Templates
+Email templates provide the framework for the store emails automatically sent to customers. These templates can be edited within the store's control panel and can also be edited locally and uploaded to a store via WebDav. For information on which user actions trigger these emails to be sent to a customer, see [Email Template Files and Snippets](https://support.bigcommerce.com/s/article/Customizing-Emails#template-files).
+
+## Accessing and Editing Email Templates
+To access and edit BigCommerce email templates: 
+
+1. Connect to your store via the WebDav client of your choice. For more on connecting to a store via WebDav, see [File Access (WebDav)](https://support.bigcommerce.com/s/article/File-Access-WebDAV). 
+
+2. Download the base templates from the /email_templates/ folder.
```

Comment:
Suggestion:
Wrap the folder name in backticks so it stands out as a folder location.